contrib/curl/include/curl/curlbuild.h

```diff
@@ -288,7 +288,7 @@
 #  define CURL_SIZEOF_CURL_SOCKLEN_T 4
 
 #elif defined(__LCC__)
-#  if defined(__e2k__) /* MCST eLbrus C Compiler */
+#  if defined(__MCST__) /* MCST eLbrus Compiler Collection */
```

Comment:
Looks like cURL uses this macro as well in the current commit (as of writing) https://github.com/curl/curl/blob/a9a8bad3ed74f5cf177d45d61a6f5d2fa2265979/include/curl/system.h#L123 +1

Reply:
> Looks like cURL uses this macro as well in the current commit (as of writing) https://github.com/curl/curl/blob/a9a8bad3ed74f5cf177d45d61a6f5d2fa2265979/include/curl/system.h#L123 +1

Yep, this is my backport patch from Curl upstream